### PR TITLE
Improve getting started

### PR DESCRIPTION
**Pull request by RCheesley**
Adding more information and screenshots to getting started documentation

### CODE REVIEW

**Comment by RCheesley** [comment]
This seems to have pulled in a PR that I made to my fork from master to update my fork ... not sure how to remote that! :blush:

File: en/setup/getting_started.md [hunk 1]
```diff
@@ -4,11 +4,77 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+###Permission errors
+If you see errors when you try to access the installation as below:
+
+![](/setup/media/permissions-error.png)
+
+the problem is likely to be due to the permissions on your files and folders.  
+
+If you have access to the command line, you can run the following commands to reset the file and folder permissions.  Make sure that you are in the directory where Mautic has been installed (using the example above, that would mean executing these commands within /m in your web root folder - you can get there using the `cd` command)
+
+```find . -type d -exec chmod 755 {} \;
```
Comment (escopecz) [inline en/setup/getting_started.md]: I believe the 3 starting and ending backticks must be on a single line so the Markdown parser would style the content in between as a code sample.

What's wrong with this PR, Ruth? What shouldn't be part of this PR?

Comment (RCheesley) [reply en/setup/getting_started.md]: screenshot-github com-2019 09 02-10_07_41 I see this on the PR which I don't think should be there (it was me bringing my fork up to date with the upstream repo).

In regards the backticks I was following https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code so I guess I need to add a line break after the final line?

Comment (escopecz) [reply en/setup/getting_started.md]: Yes, there should be a line break.

I don't think the merge commit harms anything. I refresh the master branch to ensure it is exactly the same as the remote one like this:
`$ git fetch upstream`
`$ git reset --hard upstream/master`
⚠️ but it will delete all changes and commits that are not in the remote master branch.

**Comment by aspiers** [comment]
@RCheesley Assuming you have the official mautic repo set up as a git remote called `upstream` and your own github fork as a remote called `mine`, this should work:

    git rebase upstream/master improve-getting-started
    git push -f mine improve-getting-started

However depending on how you set it up, probably one of those remotes is actually called `origin` and the other will be called something else, so amend the above commands accordingly.

I am a former git developer and have also done an unhealthy amount of work with GitHub, so you are very welcome to ping me if you ever get stuck with this kind of stuff - I'd be glad to help.

File: en/setup/getting_started.md (hunk 1 above)
Comment (aspiers) [reply en/setup/getting_started.md]: You are right that the merge commit shouldn't harm anything.  However it is nicer to avoid it if possible since it does make the history more complicated to look at.  I provided a suggestion below for using rebase to get rid of the merge.

File: en/setup/getting_started.md [hunk 2]
```diff
@@ -4,11 +4,77 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+find . -type f -exec chmod 644 {} \;
+chmod -R g+w app/cache/
+chmod -R g+w app/logs/
+chmod -R g+w app/config/
+chmod -R g+w media/files/
+chmod -R g+w media/images/
+chmod -R g+w translations/
+chown -R $USER:www-data .```
```
Comment (aspiers) [inline en/setup/getting_started.md]: For this kind of code block, I would suggest avoiding use of triple backticks and instead simply indenting the whole thing by 4 columns:

    This is some normal text.

        find . -type f -exec chmod 644 {} \;
        chmod -R g+w app/cache/

    and normal text again.

This will achieve exactly the same rendering effect, but has the advantage of making the non-rendered plain-text file much more readable.  This adheres to Markdown's original design goal, as stated on [the homepage](https://daringfireball.net/projects/markdown/):

> The overriding design goal for Markdown’s formatting syntax is to make it as readable as possible.

However, if you do use triple backticks, bear in mind the following:

- [Any text immediately following the opening backticks on the first line is used by GitHub to indicate the coding language of the rest of the block, so that it can syntax highlight correctly](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown)
- For single-line code blocks, single backticks are sufficient.  Triple backticks are only needed when the content contains backticks of its own.

**Comment by RCheesley** [comment]
Thanks for all the suggestions! Learning so much!

File: en/setup/getting_started.md [hunk 3]
```diff
@@ -4,11 +4,78 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+
+```find . -type d -exec chmod 755 {} \;
+find . -type f -exec chmod 644 {} \;
+chmod -R g+w app/cache/
+chmod -R g+w app/logs/
+chmod -R g+w app/config/
+chmod -R g+w media/files/
+chmod -R g+w media/images/
+chmod -R g+w translations/
+chown -R $USER:www-data .
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
```

File: en/setup/getting_started.md [hunk 4]
```diff
@@ -4,11 +4,78 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+
+### Email configuration
+
+The next step allows you to configure the way in which email is sent from your Mautic instance.  
+
+![](/setup/media/email-config.png)
+
+If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials and authentication mode for the systems you wish to use.
+
+###Admin login
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
### Admin login

```

File: en/setup/getting_started.md [hunk 5]
```diff
@@ -4,11 +4,78 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+Now you have completed the setup process, you will be directed to the back-end login screen at mydomain.com/s/login - this is where you log into Mautic to manage your instance, so it's worth bookmarking the login page for future reference.
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
Now you have completed the setup process, you will be directed to the back-end login screen at `http://mydomain.com/s/login` - this is where you log into your Mautic to manage your instance, so it's worth bookmarking the login page for future reference.
```

File: en/setup/getting_started.md [hunk 6]
```diff
@@ -4,11 +4,78 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
...
+
 
 ## Step 2: Add Cron Jobs
 
-Once you've installed Mautic you will need to create a few standard cron jobs to have your software process various tasks. These cron jobs can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the cron jobs you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed.)
+Once you've installed Mautic you will need to create a few standard cron jobs to have your software process various tasks. These cron jobs can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the cron jobs you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
Once you've installed Mautic you will need to create a few standard [cron jobs][cron] to have your software process various tasks. These [cron jobs][cron] can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the [cron jobs][cron] you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)

[cron][cron_jobs.md]

```

### Question

Is `[cron][cron_jobs.md]` or `[cron][cron_jobs.html]` the correct link here?  ie does `gitbook` translate a `.md` file link to the post-rendered `.html`

File: en/setup/getting_started.md [hunk 7]
```diff
@@ -26,16 +93,30 @@ Review [Cron Jobs](./../setup/cron_jobs.html) for more information on these and
...
+        m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','http://mydomain.com/m/mtc.js','mt');
 
-``` <script> (function(w,d,t,u,n,a,m){w['MauticTrackingObject']=n; w[n]=w[n]||function(){(w[n].q=w[n].q||[]).push(arguments)},a=d.createElement(t), m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m) })(window,document,'script','http(s)://yourmautic.com/mtc.js','mt'); mt('send', 'pageview'); </script> ``` 
+    mt('send', 'pageview');
+</script>
+```
 
-You'll need to change the site URL (replace yourmautic.com with your own URL) in the above script.
+You will need to change the site URL (replace mydomain.com/m with the URL to your Mautic instance) in the above script.
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
You will need to change the site URL (replace example.com/mautic with the URL to your Mautic instance) in the above script.
```

File: en/setup/getting_started.md [hunk 8]
```diff
@@ -26,16 +93,30 @@ Review [Cron Jobs](./../setup/cron_jobs.html) for more information on these and
...
 
-After installation and setup of the cron jobs you're ready to begin tracking contacts. You will need to add a simple javascript to the websites for each site you wish to track via Mautic. This is a very simple process and you can add this tracking script to your website template file, or install a Mautic integration for the more common CMS platforms. Here is an example of the tracking javascript:
+After installation and setup of the cron jobs you're ready to begin tracking contacts. You will need to add a simple javascript to the websites for each site you wish to track via Mautic. This is a very simple process and you can add this tracking script to your website template file, or install a Mautic integration for the more common CMS platforms. Here is an example of the tracking javascript which you can access by clicking on 'Tracking Settings':
+
+``` <script>
+    (function(w,d,t,u,n,a,m){w['MauticTrackingObject']=n;
+        w[n]=w[n]||function(){(w[n].q=w[n].q||[]).push(arguments)},a=d.createElement(t),
+        m=d.getElementsByTagName(t)[0];a.async=1;a.src=u;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','http://mydomain.com/m/mtc.js','mt');
```
Comment (virgilwashere) [inline en/setup/getting_started.md]:
```suggestion
    })(window,document,'script','http://example.com/mautic/mtc.js','mt');
```

File: en/setup/getting_started.md (hunk 6 above)
Comment (virgilwashere) [reply en/setup/getting_started.md]: ### Answer

    [cron][cron_jobs.html]

**Comment by virgilwashere** [comment]
⚠️ PR #361 replaces this.
